src/compiler/scala/tools/nsc/backend/jvm/BCodeBodyBuilder.scala

```diff
@@ -995,7 +995,7 @@ abstract class BCodeBodyBuilder extends BCodeSkelBuilder {
             case _ =>
               // could add some guess based on types of primitive args.
               // or, we could stringify all the args onto the stack, compute the exact size of
-              // the stringbuffer.
+              // the StringBuffer.
```

Comment:
Probably it would be a `StringBuilder`... or, it's a string buffer. (The comment is speculative.)